backend/.env

```
@@ -0,0 +1,19 @@
+# API Server
+HOST=0.0.0.0
+PORT=5000
+
+# Database
+DB_NAME="test_db"
+DB_ACCOUNT="test_account"
+DB_PASSWORD="test"
+DB_HOST="localhost"
+DB_PORT=8529
+
+# Threads
+ARBITER_THREAD=1
+WORKER=2
+
+# Auth
+APPROVEAPI_TEST_KEY=sk_test_32dqMrJgaNdm5DpOJm5bHE
+APPROVEAPI_LIVE_KEY=""
+REDIRECT_URL="https://ideaDog.site"
```

ideadog.site*, no uppercase D

```suggestion
REDIRECT_URL="https://ideadog.site"
```